cypress/fixtures/selectors.json

```diff
@@ -1,30 +1,30 @@
 {
-  "clearValues": ".react-select__clear-indicator",
-  "disabledCheckbox": "#cypress-container-single #cypress-single__disabled-checkbox",
-  "firstMultiValueRemove":
-    "#multi-select .react-select__multi-value__remove:first",
-  "groupColor": "#cypress-container-single #cypress-single-grouped .react-select__group",
-  "menuGrouped": "#grouped-options-single .react-select__menu",
-  "menuMulti": "#multi-select .react-select__menu",
+  "singleBasic": "#cypress-single",
+  "singleBasicSelect": "#basic-select-single",
+  "singleClearable": "#cypress-single-clearable",
+  "singleClearableSelect": "#clearable-select-single",
+  "singleGroupedSelect": "#grouped-options-single",
+
+  "checkboxDisable": ".disable-checkbox",
+  "checkboxEscapeClearsValue": ".escape-clears-value-checkbox",
+
+  "groupHeading": ".react-select__group-heading",
+  "indicatorClear": ".react-select__clear-indicator",
+  "indicatorDropdown": ".react-select__dropdown-indicator",
+  "menu": ".react-select__menu",
   "menuOption": "[role='option']",
+  "noOptionsValue": ".react-select__menu-notice--no-options",
+  "placeholder": ".react-select__placeholder",
+  "singleValue": ".react-select__single-value",
+
   "menuSingle": "#basic-select-single .react-select__menu",
+  "singleSelectSingleInput": "#react-select-basic-select-single-input",
+  "toggleMenuSingle": "#basic-select-single .react-select__dropdown-indicator",
```

Comment:
These are technically some old ones that I think should go away, but I kept them around because they're used in a `skip`ed test.  If we remove that test, we can remove these three selectors.